docs/core/diagnostics/faq-dumps.yml

```diff
@@ -11,15 +11,48 @@ sections:
-        answer: | 
-          On Linux-based systems, `/proc/sys/kernel/ptrace_scope` controls who can call `ptrace` and what processes they can trace. We recommend it being set to `1` - meaning only processes in the same ancestry chain can use traced. Any value higher than this requires elevation or might disable `ptrace` altogether.  
+          Why is dump collection failing on Linux?
+        answer: |
+          In order to implement dump collection, .NET processes spawn a child process called createdump. This child process uses the Linux API [ptrace()](https://www.man7.org/linux/man-pages/man2/ptrace.2.html) and also reads from the [/proc](https://www.kernel.org/doc/html/latest/filesystems/proc.html) filesystem to access thread and memory data that is written into the dump file. Although the API usage is allowed by the default security settings on many Linux distros, sometimes a less common security configuration will deny access. You may see output from the createdump process written on the console of the application being dumped such as:
+
+          ```
+          [createdump] The process or container does not have permissions or access: open(/proc/1234/mem) FAILED Permission denied (13)
+          ```
+
+          One reason that access may be denied is because a security sandbox intercepts the call using a [seccomp BPF filter](https://www.kernel.org/doc/html/v4.19/userspace-api/seccomp_filter.html). For applications running in a container using Open Container Initiative technology, the `seccomp` profile must allow for calls to `ptrace`. For example, `Docker` uses [containerd](https://github.com/moby/containerd) under the hood as a container runtime. When initializing, it specifies a default [seccomp profile](https://github.com/moby/moby/blob/master/profiles/seccomp/default.json) that allows `ptrace` only if the container host has a kernel version higher than 4.8 or if the `CAP_SYS_PTRACE` capability was specified on the container.
+
+          If the calls aren't intercepted then the kernel will do a variety of built-in access checks. The docs for [ptrace()](https://www.man7.org/linux/man-pages/man2/ptrace.2.html) include a detailed description near the bottom titled "Ptrace access mode checking" that describes how these are done. Accessing the /proc filesystem also uses a variation of the same ptrace access mode checking. This is an abbreviated summary of the security checks performed and some places we have discovered where access might be denied:
+
+          1. Either the calling process needs to have the same user id as the target process, or the calling process needs to have CAP_SYS_PTRACE. If neither of these is true access is denied. Since the .NET runtime doesn't do anything to change the user account when launching createdump the user ids should match and this step should succeed.
```

Comment:
```suggestion
          - Either the calling process needs to have the same user ID as the target process, or the calling process needs to have CAP_SYS_PTRACE. If neither of these is true, access is denied. Since the .NET runtime doesn't do anything to change the user account when launching createdump, the user IDs should match and this step should succeed.
```